pkg/kv/kvserver/stateloader/initial.go

```diff
@@ -117,12 +116,9 @@ func WriteInitialRangeState(
 	initialGCThreshold := hlc.Timestamp{}
 	initialMS := enginepb.MVCCStats{}
 
-	writeRaftAppliedIndexTerm :=
-		clusterversion.ClusterVersion{Version: replicaVersion}.IsActiveVersion(
-			clusterversion.ByKey(clusterversion.AddRaftAppliedIndexTermMigration))
 	if _, err := WriteInitialReplicaState(
 		ctx, readWriter, initialMS, desc, initialLease, initialGCThreshold,
-		replicaVersion, writeRaftAppliedIndexTerm,
+		replicaVersion, true, /* 22.1:AddRaftAppliedIndexTermMigration */
```

Comment:
Would you mind removing this parameter from `WriteInitialReplicaState` and the corresponding `if` condition inside it?